gpt4all-bindings/typescript/index.cc

```diff
@@ -66,6 +66,7 @@ Napi::Function NodeModelWrapper::GetClass(Napi::Env env) {
     if(GetInference() == nullptr) {
        std::cerr << "Tried searching libraries in \"" << library_path << "\"" <<  std::endl;
        std::cerr << "Tried searching for model weight in \"" << full_weight_path << "\"" << std::endl;
+       std::cerr << "Do you have runtime libraries installed?" >> std::endl;
```

Comment:
Uh, does that even work? Shouldn't it be `<< std::endl`? (not a C++ expert)

Reply:
ooops :rofl: ill fix